app/src/main/res/values/strings_no_translate.xml

```diff
@@ -88,4 +88,7 @@
     <string name="diff_link_wikitext">{{{diffLink}}}</string>
     <string name="wikiText_replace_url">[[%1$s|%2$s]]</string>
 
+    <string name="year_in_review_action_positive">Take me there</string>
+    <string name="year_in_review_text"><![CDATA[<strong>Wikipedia 2024 Year in Review</strong><br/><br/>Wikipedia is the largest knowledge resource ever assembled in the history of the world, and it’s freely available to everyone everywhere. Here’s a review of 2024 through the lens of Wikipedia.]]></string>
```

Comment:
(I made these strings non-translatable on purpose, since this feature will be available for English only, therefore translators don't need to be bothered.)